silence/silence.go

```diff
@@ -409,7 +421,9 @@ Loop:
 		case <-stopc:
 			break Loop
 		case <-t.C:
+			s.metrics.maintenanceTotal.Inc()
```

Comment:
should we move the counter updates to runMaintenance()?

Reply:
Good shout, I considered neglected the idea and now that you mention it - I think it's better.e